browserscripts/browser/appConstants.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  requires: { privilege: true },
+  function: function() {
+    const { AppConstants } = ChromeUtils.import(
+      'resource://gre/modules/AppConstants.jsm'
+    );
+    return {
+      constants: JSON.stringify(AppConstants)
```

Comment:
nit: `appConstants:`.  Help future us grep for this!

Reply:
Just to clarify, even though the object that we import is from `AppConstants.jsm`, we want to name the object that we retrieve `appConstants`?

Reply:
Update: I'm an idiot. Changing the case of that variable will *not* work. The 

`const { AppConstants } = ...`

destructures an object returned by the import statement and assigns the field from the returned object named `AppConstant` to our variable `AppConstant`. If those variable names differ, then there will not be a match and the variable will be null. 

TL;DR: We have to keep it named `AppConstants`.

Reply:
There are two different things going on here.  One is destructuring: that name must be `AppConstants`.  The other is what we expose, which in your patch is `{constants: ...}`.  Make that `{appConstants: ...}` or `{AppConstants: ...}` so that it's clear what set of constants these are (or, somewhat clear, at least).  `appConstants` is more in keeping with browsertime, which uses `userAgent`, etc.